src/Colopl/Spanner/Schema/Blueprint.php

```diff
@@ -131,7 +131,7 @@ public function floatArray($column)
 
     /**
      * @param string $column
-     * @param int $length
+     * @param int|string $length
```

Comment:
```suggestion
     * @param int|'MAX'$length
```

I think that `$length` accepts only 'MAX' as string.
https://phpstan.org/r/91c2b2b0-bbf1-4b66-b216-453c3bb73c7f

However, PhpStorm still cannot recognize string literal of PHPDoc, so I leave it up to you to take it in.

Reply:
Thanks, but IDE support is crucial. Let's revisit when PhpStorm supports this.